### PR TITLE
Add 'used' feature to check_swap

### PR DESCRIPTION
**Pull request by mcktr**
This implements the _used_ feature to check_swap to print the used swap
instead of the default available swap.

### CODE REVIEW

**Comment by mcktr** [comment]
Updated the ITL CheckCommand to include the new parameter.

File: itl/command-plugins-windows.conf [hunk 1]
```diff
@@ -258,6 +258,10 @@ object CheckCommand "swap-windows" {
 			value = "$swap_win_unit$"
 			description = "Unit to display swap in"
 		}
+		"-U" = {
+			set_if = $swap_win_show_used$
```
Comment (dnsmichi) [inline itl/command-plugins-windows.conf]: This breaks the config tests, missing double quotes.

**Comment by mcktr** [comment]
Fixed the config test. Tests are now all green.

**Comment by dnsmichi** [comment]
Thanks 👍

**Pull request merged**